### PR TITLE
Global comment stream close setting

### PR DESCRIPTION
**Pull request by impronunciable**
It adds a global setting for autoclosing comments. It can be configured in the admin.

Also when going to the comment stream as an admin you can see when the stream will close.

### CODE REVIEW

**Comment by wyattjoh** [comment]
Do we have a way to turn on/off auto closing?

File: client/coral-admin/src/translations.json [hunk 1]
```diff
@@ -117,6 +121,11 @@
       "community": "Comunidad",
       "closed-comments-desc": "Escribe un mensaje para cuando los comentarios se encuentran cerrados",
       "closed-comments-label": "Escribe un mensaje...",
+      "never": "Nunca",
```
Comment (gabelula) [inline client/coral-admin/src/translations.json:124]: Is the string "never" missing from the english strings?

**Review by gabelula — CHANGES_REQUESTED** [review]

File: client/coral-admin/src/translations.json (hunk 1 above)
Comment (impronunciable) [reply client/coral-admin/src/translations.json:124]: we are not using it

**Comment by impronunciable** [comment]
@wyattjoh you can leave it blank

**Comment by impronunciable** [comment]
@gabelula removed that translation and merged with master

**Review by gabelula — APPROVED** [review]

**Pull request merged**